students/rhamersky/Lesson_6/mailroom_part4.py

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python3
+# Description: This program will allow a user to input a donor and a donation.
+# The user will also be able to send out thank you notes to the donors.
+# Developer: Ryan Hamersky
+# Date: 05/13/2018
+# Rev: A - 05/16/2018 updated code to reflect comments made by the instructor in mailroom_part.py.
+#      Update variables to snake_case. Removed unnecessary 'continues' and 'returns'.
+#      Changed 'bool' variable to 'bool_flag'. Updated misleading function names.
+#      Reduced the amount of unnecessary print() statements.
+#
+# Rev: B - 05/19/2018 updated code to reflect comments made by the instructor in mailroom_part3.py.
+
+# -----Data Section-----
+import os
+str_name = str()    # --> User's name
+int_option = int()  # --> User's option.
+tup_donor_names = (["William Gates, III",1,10000.0],["Mark Zuckerberg",1,20000.0],
+                   ["Jeff Bezos",1,1200.0],["Paul Allen",1,5000.0],
+                   ["Mike Scott",1,25.0])  # --> Tuple that stores all donor's information
+str_salutation = ""   # --> Email salutation
+str_body = ""         # --> Email body
+str_valediction = ""  # --> Email valediction
+
+
+# -----Process Section-----
+#  Methods defined in the menu class are related to general menu checks or options based on user inputs.
+
+class Menu():
+    @staticmethod
+    def menu_options():
```

Comment:
If your class only includes a single static method, that's a good indication that you don't need to be using a class at all. We'll learn a lot more about classes and object-oriented Python later in the course, but generally you only need to use a class in Python if you have a collection of related data (attributes) and logic (methods) that act upon that data. 

In this case, the only reason I can see for using a class is to provide some organizational separation/hierarchy for your program, but there are more appropriate options like module level functions and modules themselves for accomplishing this in Python.